packages/core/src/client.ts

```diff
@@ -2056,7 +2056,7 @@ export class MedplumClient extends EventTarget {
     this.storage.setObject('logins', logins);
   }
 
-  private async refreshProfile(): Promise<ProfileResource | undefined> {
+  async refreshProfile(): Promise<ProfileResource | undefined> {
```

Comment:
this helped refresh the AppShell leftnav, but open to find another way to refresh the UserConfiguration

Reply:
The trick is to get `<MedplumProvider>` to re-render.

Here is where `<MedplumProvider>` listens for change events: https://github.com/medplum/medplum/blob/main/packages/react/src/MedplumProvider/MedplumProvider.tsx#L37-L47

Here is background reading on React Context: https://legacy.reactjs.org/docs/context.html